SpliceAIWeb/Dockerfile

```diff
@@ -12,5 +18,13 @@ RUN pip install --no-cache-dir -U pip \
 COPY . /app
 WORKDIR /app
 
-EXPOSE 5001
-CMD ["gunicorn", "-b 0.0.0.0:5001", "-t 1200", "-w 4", "spliceai_web.app:app"]
+RUN chmod +x /app/gcsfuse_run.sh
```

Comment:
Where does this come from?

Reply:
Sorry, had forgotten to add this file to the PR. It follows the GCP example.